CONTRIBUTING.md

```diff
@@ -0,0 +1,23 @@
+# Contributing to OpenChart
+
+Thanks for your interest in contributing to OpenChart. This guide aims to outline some of the standards we've set for contributions to the project.
+
+## Coding standards
+In order to keep the standards of our codebase high, we ask that any code you write adheres to the the [C# Coding Conventions](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/inside-a-program/coding-conventions). This helps to keep everything consistent and readable. 
```

Comment:
Probably slim this down to one sentence

"In order to keep the codebase consistent and easy to maintain, we ask that you please adhere to the recommended <C# Coding Conventions>."